src/core/reducers/addons.js

```diff
@@ -172,6 +172,8 @@ export function createInternalAddon(
         updateURL: apiAddon.theme_data.updateURL,
         version: apiAddon.theme_data.version,
       }),
+      // We should be using this property.
+      themeData: removeUndefinedProps({ ...apiAddon.theme_data }),
```

Comment:
This can just be `themeData: apiAddon.theme_data` because `removeUndefinedProps()` was only necessary to make the spread not override existing property values with `undefined`.